src/App.jsx

```diff
@@ -14,45 +16,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const GOOD_LEGTH = 'length';
+const GOOD_NAME = 'name';
+
+function getPreparedGoods(goods, sortField, isReverse) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case GOOD_LEGTH:
+          return good1.length - good2.length;
+        case GOOD_NAME:
+          return good1.localeCompare(good2);
+        default:
+          return 0;
```

Comment:
In this case, you will call the sort method even if it's not needed. Consider moving switch...case outside of the sort callback